CHANGELOG.rst

```diff
@@ -7,8 +7,27 @@ All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
 
-[Unreleased 1.1.9] - `master`_
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+[Unreleased 1.1.10] - `master`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Added
+-----
+
+
+Changed
+-------
+
+
+Removed
+-------
+
+
+Fixed
+-----
+
+
+[1.1.9] - 2019-01-08
```

Comment:
I believe the other release dates are using YYYY-MM-DD !

Reply:
Thanks for the catch! Sick brain doesn't work so well today 🤦‍♀

Reply:
👍 👍